tests/testsuite/build_script.rs

```diff
@@ -5165,3 +5165,70 @@ fn custom_build_closes_stdin() {
         .build();
     p.cargo("build").run();
 }
+
+#[cargo_test]
+fn test_both_two_semicolons_and_one_semicolon_syntax() {
```

Comment:
If the existing tests were updated to `cargo::`,  we should have tests focused on `cargo:` syntax to ensure we don't break support

Reply:
Updated.